### ISSUE TITLE
Add CEQR Type II source data to data library and implement initial dbt script in `ceqr-survey`

### ISSUE THREAD

**Issue by damonmcc** [NYCPlanning/data-engineering#575]
Source Data Review doc [here](https://nyco365.sharepoint.com/:x:/r/sites/NYCPLANNING/itd/edm/_layouts/15/Doc.aspx?sourcedoc=%7BBAE15377-87A6-4188-9637-36A89AC1DF98%7D&file=CEQR_Type_II_Data_Source_Review.xlsx&action=default&mobileredirect=true)

for datasets where Status = "approved", we can start adding recipes and extracting data

- [x] Zoning
- [x] E-Designation
- [x] Air
- [x] Noise

**Comment by fvankrieken** [comment]
For simplicity, seems to make sense to limit this issue to the datasets needed for the demo - categories zoning, air quality, and noise

For datasets that correspond to CEQR App, we've elected to archive to data library the outputs that we've produced in `edm-publishing/ceqr-app-data`. Longer term it'd be a lot nicer to archive raw data and process as part of extract code, but that'll need to wait till we really get into #484 

In addition to getting these datasets in `edm-recipes`, they should each have a corresponding script in `products/ceqr/models/intermediate/`. This script should pull in the dataset (imported via dcpy recipe/load logic) using dbt's `source` function, then format the table so that it can be easily joined to pluto bbls to find violations.
- table should have 3 columns: 
  - "variable" - corresponds to dataset (see "Variable" column in data source excel)
  - "id" - some unique identifier of row within its dataset
  - "geom" - the geometry of the dataset buffered as specified in "Data Processing" column in data source excel
